cmd/edit/cluster/cmd.go

```diff
@@ -192,6 +203,29 @@ func run(cmd *cobra.Command, argv []string) {
 		reporter.Errorf("Failed to get AWS creator: %v", err)
 		os.Exit(1)
 	}
+	awsCredentials := &clusterprovider.AWSCredentials{}
+	if cmd.Flags().Changed("update-aws-credentials") {
+		if confirm.Confirm("update AWS credentials for cluster %s", clusterKey) {
+			ok, err := awsClient.ValidateCredentials()
+			if err != nil {
+				reporter.Errorf("Error validating AWS credentials: %v", err)
+				os.Exit(1)
+			}
+			if !ok {
+				reporter.Errorf("AWS credentials are invalid")
+				os.Exit(1)
+			}
+			reporter.Infof("AWS credentials are valid!")
+			credValue, err := awsClient.GetCredentials()
```

Comment:
Actually, hold on. We should be sending the credentials form the cloudformation stack, instead of the user's. I think you should call `awsClient.GetAccessKeyFromStack()` instead.

Note that once #118 is merged this will change to `awsClient.GetAWSAccessKeys()`

Reply:
@pvasant can you rebase and change this to use `awsClient.GetAWSAccessKeys()`?